Modules/readline.c

```diff
@@ -1241,6 +1244,10 @@ setup_readline(readlinestate *mod_state)
     else
         rl_initialize();
 
+    if (!using_libedit_emulation) {
+        rl_variable_bind ("enable-bracketed-paste", "off");
```

Comment:
Can you please move this line into a subfunction and also use it in readline_read_init_file_impl()?

I would like to see a command explaining why Python disables explicitly the bracketed paste mode, whereas at the first look, it looks like a good idea to enable it by default. For example, explain that the REPL doesn't support it, and it injects "\x1b[?2004h" escape sequence which causes test failures. See my commands at https://bugs.python.org/issue42819 Also mention "bpo-42819" in your comment.